packages/ui/src/machines/authenticator/actions.ts

```diff
@@ -154,6 +155,12 @@ export const handleBlur = assign({
   },
 });
 
+export const resendCode = async (context) => {
```

Comment:
Why does this need to be async if there is no `await`?

Reply:
I'll add the await here!

Reply:
[5f8bbaa](https://github.com/aws-amplify/amplify-ui/pull/3201/commits/5f8bbaabd2d2a81c3ff5ddcbd243ffbdd5db3c86)